src/leases/components/search/Search.js

```diff
@@ -236,6 +236,27 @@ class Search extends PureComponent<Props, State> {
             />
           </Column>
         </Row>
+        <SearchRow>
+          <SearchLabelColumn>
+            <SearchLabel>Palvelukokonaisuus</SearchLabel>
+          </SearchLabelColumn>
+          <SearchInputColumn>
+            <FormField
+              autoBlur
+              disableDirty
+              fieldAttributes={{
+                label: 'Palvelukokonaisuus',
```

Comment:
This label is repeated in many places, so it is recommended to use the enum in `leases/enums.js`